examples/two_mass_spring_damper/run.py

```diff
@@ -0,0 +1,94 @@
+import matplotlib.pyplot as plt
+from numpy import linspace
+from pydy.system import System
+from sympy import symbols, Matrix
+from sympy.physics.mechanics import dynamicsymbols, eombase
+
+# Define the dynamic symbols
+
+x1, x2, u1, u2 = dynamicsymbols('x1 x2 u1 u2')
+
+# Define the constant symbols
+
+m1, c1, k1 = symbols('m1 c1 k1')
+m2, c2, k2 = symbols('m2 c2 k2')
+
+# Define the mass matrix and forcing vector
+
+mm = Matrix([[m1,  0],
+             [0,  m2]])
+f = Matrix([-(k1+k2)*x1 + k2*x2 - (c1+c2)*u1 + c2*u2,
+            k2*x1 - k2*x2 + c2*u1 - c2*u2])
+
+mm_full = Matrix([[1, 0,  0,  0],
+                  [0, 1,  0,  0],
+                  [0, 0, m1,  0],
+                  [0, 0,  0, m2]])
+f_full = Matrix([u1,
+                 u2,
+                 -(k1+k2)*x1 + k2*x2 - (c1+c2)*u1 + c2*u2,
+                 k2*x1 - k2*x2 + c2*u1 - c2*u2])
+
+# Form the rhs of q' = G(q, u, t, r, p). Kinematic equation
+
+G = Matrix([u1, u2])
```

Comment:
I agree that lagrange's method uses q, q' and q'' and doesn't even store a "speeds" variable (`LagranesMethod.u` returns `LagrnagesMethod._qdots`). This results in `Derivative(q(t), t)` potentially showing up in `L.mass_matrix` and `L.forcing`. I suppose these would need to be replaced with `u`'s so that no derivative terms show up in either `L.mass_matrix` or `L.forcing`. Leaving the equations in terms of q'' does not have any other functional difference that I can think of because `LagrangesMethod.mass_matrix_full` and `LagrangesMethod.forcing_full` specify that u=q' and u' or q'' doesn't actually show up in the resulting symbolics.